data/trie/interceptedNode.go

```diff
@@ -17,8 +18,10 @@ var _ process.InterceptedData = (*InterceptedTrieNode)(nil)
 // InterceptedTrieNode implements intercepted data interface and is used when trie nodes are intercepted
 type InterceptedTrieNode struct {
 	node           node
-	SerializedNode []byte
+	serializedNode []byte
 	hash           []byte
+
+	mutex sync.RWMutex
```

Comment:
could have been aligned to the other fields and L23 could have been removed.

Reply:
Done.